libdnf5/base/goal.cpp

```diff
@@ -605,6 +605,10 @@ GoalProblem Goal::Impl::resolve_group_specs(std::vector<GroupSpec> & specs, base
     comps::EnvironmentQuery base_environments_query(base);
     for (auto & [action, reason, spec, settings] : specs) {
         bool skip_unavailable = settings.resolve_skip_unavailable(cfg_main);
+        if (action == GoalAction::REMOVE) {
+            // do not treat the removal of a not-installed group as an error
+            skip_unavailable = true;
+        }
```

Comment:
This is minor. The call `settings.resolve_skip_unavailable(cfg_main);` will incorrectly set that for the request was used some setting, but if it is a remove action then it is ignored.

What about to do something like: 
`bool skip_unavailable = action == GoalAction::REMOVE ? true : settings.resolve_skip_unavailable(cfg_main);`

Reply:
Right, fixed.